manual_cases.md

```
@@ -0,0 +1,533 @@
+| #  | ID                      | Title                                                                                                    | Importance | Steps |
+|----|-------------------------|----------------------------------------------------------------------------------------------------------|------------|-------|
+| 1  | [OCP-87356](#ocp-87356) | [v2]Verify that filtered operator catalogs can be successfully mirrored to Nexus registry without manifest conversion errors | Medium     | 3     |
+| 2  | [OCP-82938](#ocp-82938) | [OCPBUGS-57461] m2d twice, then d2m in an air-gapped env, no error "trying to fetch from registry.redhat.io" | Medium     | 7     |
+| 3  | [OCP-79068](#ocp-79068) | [OCPBUGS-48314] - Verify oc-mirror V2 works on FIPS enabled and STIG compliant RHEL 9 system             | High       | 8     |
+| 4  | [OCP-76473](#ocp-76473) | [CLID-75] Validate oc-mirror works well with proxy                                                       | High       | 3     |
+| 5  | [OCP-75240](#ocp-75240) | [CLID-55] performance of oc-mirror has been improved with introduction of --max-parallel-download flag compared to 4.16 v2 | Low        | 3     |
+| 6  | [OCP-75219](#ocp-75219) | [CLID-5] Make sure disk2mirror no need to connect to network [v2]                                        | Critical   | 6     |
+| 7  | [OCP-73854](#ocp-73854) | [CLID-5] oc-mirror compatibility test for v2                                                             | Low        | 3     |
+| 8  | [OCP-73039](#ocp-73039) | [CLID-5] Support breakpoint resume for v2                                                                | Low        | 3     |
+| 9  | [OCP-72991](#ocp-72991) | [CLID-5] Make sure graph image works well for v2                                                         | Critical   | 4     |
+| 10 | [OCP-72923](#ocp-72923) | [CLID-5][CLID-14] full mirror setting for specified package v2 mirror                                    | High       | 2     |
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "clid-14"

(MD052, reference-links-images)